src/components/SearchAndFilter/SearchAndFilter.js

```diff
@@ -239,31 +207,48 @@ const SearchAndFilter = ({
           className="search-and-filter__panel"
           aria-hidden={filterPanelHidden}
         >
-          <ContextualMenu>
-            {searchTerm.length > 0 && (
-              <div
-                className="search-prompt"
-                onClick={() => handleSubmit()}
-                onKeyDown={(e) => searchPromptKeyDown(e)}
-                role="button"
-                tabIndex="0"
-              >
-                Search for <strong>{searchTerm}</strong>...
-              </div>
-            )}
-            {filterPanelData.map((filterPanelSectionData) => {
-              return (
-                <div key={filterPanelSectionData.id}>
-                  <FilterPanelSection
-                    data={filterPanelSectionData}
-                    toggleSelected={toggleSelected}
-                    searchData={searchData}
-                    searchTerm={searchTerm}
-                    sectionHidden={filterPanelHidden}
-                  />
-                </div>
-              );
-            })}
+          <ContextualMenu
+            autoAdjust={false}
+            className="search-and-filter__contextual-menu"
+            constrainPanelWidth
+            onToggleMenu={(visible) => setFilterPanelHidden(!visible)}
+            position="left"
+            visible={!filterPanelHidden}
+            dropdownClassName="search-and-filter__dropdown"
+          >
+            <CSSTransition
+              appear={true}
+              in={true}
+              timeout={200}
+              classNames="search-and-filter__contextual-menu"
+            >
+              <>
```

Comment:
Is the `<>` fragment needed? They all seem to be wrapped in single parent component.

Reply:
`<CSSTransition>` only allows one child. In this case we have the search term and results as the two children which need to be wrapped.